views/App.css

```diff
@@ -3,6 +3,7 @@
   --primary-main: #2196f3;
   --text-secondary: #054da7;
   --dark-transparent-grey: #616161e6;
+  --underline-grey: #0000001f;
```

Comment:
is this a valid hex code? It has 8 digits instead of 6.

Reply:
yes, I got it from a vscode extension and I thought the same so I verified it..
<img width="896" alt="Screenshot 2024-03-21 at 7 55 44 AM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/22e962ac-4528-4e3b-aaf8-24d3ce465629">